doc/source/ray-dashboard.rst

```diff
@@ -0,0 +1,157 @@
+Ray Dashboard
+==============
+Welcome to the Ray dashboard page! 
+
+Ray's built-in dashboard provides metrics, charts, and other features that help Ray users to understand Ray clusters and libraries.
+This includes cluster node level metrics, actor level metrics, configurations, profiling & killing actor buttons, and Ray library metrics such as Tune metrics. 
+Check out the documentation to understand a rich set of features we offer.
+
+Getting Started
+---------------
+  
+Access Dashboard
+~~~~~~~~~~~~~~~~
+If you start Ray on your laptop, dashboard is running by default, and you can access it through its URL. 
+By default the URL is `localhost:8265`. Note that the port will be changed if the default port is not available.
+
+The URL is printed when Ray starts up.
+
+.. code-block:: text
+
+  INFO resource_spec.py:212 -- Starting Ray with 4.2 GiB memory available for workers and up to 2.12 GiB for objects. You can adjust these settings with ray.init(memory=<bytes>, object_store_memory=<bytes>).
+  INFO services.py:1093 -- View the Ray dashboard at localhost:8265
+
+Dashboard is also available when using autoscaler. Check out `Monitoring Autoscaler Status <autoscaling.html#monitoring-cluster-status>`_ for more details.
+
+Example Usage
+-------------
+
+To understand each components, look at `Dashboard Components <ray-dashboard.html#components>`_
+
+Debugging a Blocked Actor
+~~~~~~~~~~~~~~~~~~~~~~~~~
+If creating an actor requires resources (e.g., CPUs, GPUs, or other custom resources) that are not currently available, the actor cannot be created until those resources are added to the cluster or become available.
+This can cause an application to hang. To alert you to this issue, infeasible tasks are shown in red in the dashboard, and pending tasks are shown in yellow.
+You can find this information from the `Logical View` tab. 
+
+- **Infeasible Actor Creation** happens when an actor requires more resources than a Ray cluster can provide.
```

Comment:
I'd remove this

Reply:
Why would you want to remove this definition? Do you think these two are clear enough to distinguish without extra explanation?